### ISSUE TITLE
Linear schedule for learning rate

### ISSUE THREAD

**Issue by Piggelinus** [ikostrikov/pytorch-a2c-ppo-acktr#31]
The ACKTR paper describes "Both the baseline (A2C) and our method used a linear schedule for the learning rate over the course of training."
Is this implemented? I cannot find it in the code.

**Comment by ikostrikov** [comment]
In my experience A2C worked better without a linear schedule.

Also it's the default option in the OpenAI repo: 
https://github.com/openai/baselines/blob/master/baselines/a2c/run_atari.py#L37

However, it's easy to add a linear schedule in pytorch:
http://pytorch.org/docs/master/optim.html#torch.optim.lr_scheduler.LambdaLR

**Comment by Piggelinus** [comment]
Great!
I'll have a look to see if I can add it myself.
Will send a pull request if it works out.

**Comment by ikostrikov** [comment]
At some point I want to change KFAC to be a wrapper of a model instead of a separate optimizer (since it can be used in combination with other optimizers anyway).

Do you need to use ACKTR specifically?

**Comment by Piggelinus** [comment]
I'm currently using all 3 of the agents but will likely move on to only using ACKTR soon.

I've now added a simple way of decreasing the learning similar to the adjust_learning_rate function
from this example https://github.com/pytorch/examples/blob/master/imagenet/main.py
It's only for a linear schedule and doesn't use the torch.optim.lr_scheduler.LambdaLR.
